sites/fast-website/src/app/components/fast-frame/fast-frame.template.ts

```diff
@@ -114,22 +128,34 @@ export const FastFrameTemplate = html<FastFrame>`
                                 @change="${(x, c) =>
                                     x.saturationChangeHandler(c.event as CustomEvent)}"
                             >
-                                <div slot="track" class="saturation-slider-track" style="background-image: linear-gradient(to right, ${x =>
-                                    hslToRGB(
-                                        new ColorHSL(x.hue, 0, x.lightness)
-                                    ).toStringHexRGB()}, ${x =>
-    hslToRGB(new ColorHSL(x.hue, 1, x.lightness)).toStringHexRGB()});"></div>
+                                <div
+                                    slot="track"
+                                    class="saturation-slider-track"
+                                    style="background-image: linear-gradient(to right, ${x =>
+                                        hslToRGB(
+                                            new ColorHSL(x.hue, 0, x.lightness)
+                                        ).toStringHexRGB()}, ${x =>
+                                        hslToRGB(
+                                            new ColorHSL(x.hue, 1, x.lightness)
+                                        ).toStringHexRGB()});"
+                                ></div>
                             </fast-slider>
                         </div>
                     </div>
                 </fast-tab-panel>
-                <fast-tab-panel id="style-tab-panel" class="${x =>
-                    x.expanded ? "tab-panel-expanded" : ""}">
+                <fast-tab-panel
+                    id="style-tab-panel"
+                    class="${x => (x.expanded ? "tab-panel-expanded" : "")}"
+                >
                     <div class="content">
-                        <h1><span class="content-heading-highlight">Fast Frame</span> Styles</h4>
+                        <h4>
+                            <span class="content-heading-highlight">Fast Frame</span>
+                            Styles
+                        </h4>
                         <h2>Adjust style settings on the fly</h2>
```

Comment:
Is this `h2` correct given the above change from `h1` to `h4`?

Reply:
This would continue to be an issue, let’s rake a full look across the entire page and ensure proper DOM hierarchy. There should be online tools that will transfer your markup to a more readable outline.

Reply:
@chrisdholt are there any specific tools you recommend? I found some chrome extensions that deal with headings, but not sure if they are doing the right job.

Reply:
I’d have to dig in here, and likely won’t have time this morning. I’d try to find a tool that outlines the document.

Reply:
Headings also will differ depending on where they are used.  It may be worthwhile investigating programmatic solutions such that inheritance determines they heading values used in the controls.  Also, it's confusing if you jump from H2 to H4 in a screen reader.  Headings should be contextual based on content, not styles.  In fact, the easiest way to implement correct heading structure is to turn off stylesheets together. Then come back through and style as needed once the semantic headings structure is correctly represented.

Reply:
Agree with the above in general, but not sure for the homepage a systematic approach is required or needed right now. I’d be happy with the immediate fixes for the correct semantics and structure. @khamudom, @awentzel details some helpful stuff in the above comment on landing and testing accurate structure.

Reply:
What I've noticed is when I am running a heading check extension, it doesn't find any other headings. For example, the headings in the `fast-frame` component are not getting checked as part of the website. But while taking a look at the `fast-frame` template, I think we can update the <h4> and <h2> tags to be a <p> tag. I did some testing and this seems to work when I also update the style selector.